azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -108,6 +110,10 @@ public void init(final ServletConfig config) throws ServletException {
     this.userManager = server.getUserManager();
     this.projectManager = server.getProjectManager();
     this.executorManagerAdapter = server.getExecutorManager();
+    if (this.executorManagerAdapter instanceof ContainerizedDispatchManager) {
+      this.containerizedImpl =
+          ((ContainerizedDispatchManager) this.executorManagerAdapter).getContainerizedImpl();
```

Comment:
Why not add "getContainerizedImpl()" to the "ContainerizedImpl" interface?

Reply:
ContainerizedDispatchManager does not implement ContainerizedImpl interface.